apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx

```diff
@@ -321,73 +338,159 @@ const PlanUpdateSidePanel = () => {
+                            (prev, cur) => prev + cur.total_price,
+                            0
+                          )
+                        ) ?? 0}
+                      </Table.td>
+                    </Table.tr>
+                  </>
                 }
-              />
+              ></Table>
 
-              {subscriptionPreview.number_of_projects !== undefined &&
-                subscriptionPreview.number_of_projects > 1 && (
-                  <p className="text-sm mt-2">
-                    All {subscriptionPreview.number_of_projects} projects from your organization "
-                    {selectedOrganization?.name}" will use the new {planMeta?.name} plan.
-                  </p>
-                )}
+              <InformationBox
+                className="mt-4"
+                title="Usage-billing for Compute"
+                icon={<IconInfo />}
+                defaultVisibility={true}
+                hideCollapse={true}
+                description={
+                  <div>
+                    <p className="text-sm mt-2">
+                      Each project is a dedicated server and database. Paid plans come with $10 of
+                      Compute Credits to cover one project on the default Starter Compute size or
+                      parts of any compute addon. Additional unpaused projects on paid plans at
+                      least incur compute usage costs starting at $10 per month, billed hourly.
+                    </p>
+
+                    {subscription?.plan?.id === 'free' && (
+                      <p className="text-sm mt-2">
+                        Mixing paid and non-paid projects in a single organization is not possible.
+                        If you want projects to be on the free plan, use self-serve project
+                        transfers.
+                      </p>
+                    )}
+
+                    <div className="space-x-3 mt-2">
+                      <Button asChild type="default" icon={<IconExternalLink strokeWidth={1.5} />}>
+                        <Link
+                          href="https://supabase.com/docs/guides/platform/org-based-billing"
+                          target="_blank"
+                          rel="noreferrer"
+                        >
+                          How billing works
+                        </Link>
+                      </Button>
+                      {subscription?.plan?.id === 'free' && (
+                        <Button
+                          asChild
+                          type="default"
+                          icon={<IconExternalLink strokeWidth={1.5} />}
+                        >
+                          <Link
+                            href="https://supabase.com/docs/guides/platform/project-transfer"
+                            target="_blank"
+                            rel="noreferrer"
+                          >
+                            Project Transfers
```

Comment:
just to keep the button casings consistent
```suggestion
                            Project transfers
```